cernopendata/modules/theme/assets/semantic-ui/js/search/components/DefaultListItem.js

```diff
@@ -33,7 +33,7 @@ import { stripHtml } from "../utils";
 const CODefaultListItem = ({ result: { metadata, id }, index }) => {
   const link = `/record/${id}`;
   return (
-    <Item key={index}>
+    <Item key={index} className="results-item">
```

Comment:
I have a doubt, it might be something I don't know. I can't see this class (`results-item`) in the output HTML, however, the changes are applied (there is no `text-decoration`). Could it be that the change is applied by the other CSS changes you did? Either that or I am missing something.

Reply:
Hm, the class should be there. Here is what my browser shows me:
![Screenshot from 2021-02-05 14-55-49](https://user-images.githubusercontent.com/4990025/107042731-5d34f480-67c2-11eb-977b-247456fb2868.png)

Reply:
I can see it in an incognito window, thanks for double-checking!